src/tasks/checker/checker-v2/index.ts

```diff
@@ -46,7 +46,6 @@ export async function main(): Promise<void> {
     useDefaultPAEndpoint: parameterMap['UseDefaultPACheckerEndpoint'],
     customPAEndpoint: parameterMap['CustomPACheckerEndpoint'],
     geoInstance: { name: "GeoInstance", required: false, defaultValue: undefined },
-    saveResults: parameterMap['SaveResults'],
```

Comment:
saveResults is a required parameter, make it optional or pass default value i.e. false to it.

Reply:
Ack